Formula/odo.rb

```diff
@@ -19,6 +19,8 @@ class Odo < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "23644efe576abf9c3e3a469cf1baad05b3a3cde749998045b2992c290cf57884"
   end
 
+  conflicts_with "odo-dev", because: "odo-dev also ships 'odo' binary"
```

Comment:
Let's add the `conflicts_with` after this PR is merged to avoid wastefully rebuilding this formula. We'll need to remove `conflicts_with` from `odo-dev` too.

Reply:
ok, I'll remove it from this PR.